book/modules/numeric.py

```diff
@@ -0,0 +1,50 @@
+from django.db.models   import (
+        Sum,
+        Q,
+)
+
+from user.models import UserBook
+
+
+def get_reading_numeric(book_id):
+    user_books = UserBook.objects.select_related('book').filter(book_id=book_id)
+
+    if not user_books.exists():
+        return {'message':'NOT_EXIST'} 
+
+    # 완독할 확률   = 책 완독한 독자/책 전체 독자 * 100 (완독여부는 읽은 page/책 총 page)
+    # 완독 예상시간 = 책 완독자 총 reading time / 책 완독자 수 
+    finish_readers = red_time = 0
```

Comment:
```suggestion
    finish_readers = red_time = 0
```
```suggestion
    finish_readers = read_time = 0
```
혹시 읽은 시간을 담는 변수라면 오타 확인해주세요 :)
이 뜻이 아니라면 혹시, 어떤 의미를 담은 변수일까요?

Reply:
앗, 이미 읽은 시간이라고 생각해서 과거형으로 작성했습니다 ^^;

Reply:
아하 그럼 `read`가 더 맞을 것 같습니다 :) 현재, 과거 모두 단어가 같은 단어라서용!

Reply:
아아 순간 과거형 발음은 red 에 가까우니 red인줄 알았네요. 수정하겠습니다.